packages/boot/test/bootstrapTests/orchestration.test.ts

```diff
@@ -305,7 +305,11 @@ test.serial('revise chain info', async t => {
   );
 
   const hotchain = await EV(agoricNames).lookup('chain', 'hot');
-  t.deepEqual(hotchain, { allegedName: 'Hot New Chain', chainId: 'hot-1' });
+  t.deepEqual(hotchain, {
```

Comment:
not worth a revision but I think this "chore: ChainInfo tests include namespace and reference" should be "test:"